### PR TITLE
remove dconf access

### PR DESCRIPTION
**Pull request by dgcampea**

### CODE REVIEW

**Comment by antenore** [comment]
Hey @dgcampea Thank you!
LGTM, what do you think @larchunix ?

**Comment by TingPing** [comment]
The commit message could be cleared up, there is no portal.

File: org.remmina.Remmina.json [hunk 1]
```diff
@@ -42,9 +42,7 @@
         "--talk-name=org.gnome.Shell",
         /* Local network host discovery */
         "--system-talk-name=org.freedesktop.Avahi",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+        "--metadata=X-DConf=migrate-path=/org/remmina/Remmina/"
```
Comment (TingPing) [inline org.remmina.Remmina.json]: I saw you blindly add this to another PR. Does this actually use settings at this path?

Comment (dgcampea) [reply org.remmina.Remmina.json]: now that I checked with dconf editor, there seems to be no traces of remmina anywhere. Are these permissions (and the metadata line) unnecessary then?

**Comment by dgcampea** [comment]
settings (rdp hosts, remmina) seem to be preserved after removing dconf access.

**Comment by TingPing** [comment]
> settings (rdp hosts, remmina) seem to be preserved after removing dconf access.

No need to guess: `dconf dump /org/remmina/Remmina/` If that lists something you need to migrate it.

**Comment by antenore** [comment]
I don't follow you anymore guys, sorry :-D 

Please tell me when and if I can merge it.

Testing the flatpak works properly for me.

Thank you again!

CC @TingPing @larchunix

**Comment by TingPing** [comment]
@antenore It looks good now! Sorry about the noise just making sure the change was understood.

**Comment by antenore** [comment]
@TingPing no problem at all! Thank you!

**Pull request merged**